app/models/user.rb

```diff
@@ -6,9 +6,6 @@ def validate(record)
   end
 end
 
-# Overwrites authlogic username regex to allow one character usernames
-Authlogic::Regex::LOGIN = /\A[A-Za-z\d_\-]*\z/
```

Comment:
@jywarren is this line safe to remove?

Reply:
I would say probably not - i think we need that? Can we look at the pull request that generated that line, using the "blame" view of that file on GitHub?

Thank you!

Reply:
@jywarren @VladimirMikulic Have updated the pr keeping in mind the changes.Thanks :)